t/09-authn-request.t

```diff
@@ -56,4 +60,97 @@ ok($signed);
 
 my $verify = $signer->verify($signed);
 ok($verify);
+
+$ar = Net::SAML2::Protocol::AuthnRequest->new(
+    issuer        => 'http://some/sp',
+    destination   => 'http://some/idp',
+    nameid_format => 'urn:oasis:names:tc:SAML:2.0:nameid-format:persistent',
+    nameid_allow_create => 1,
+    force_authn   => '1',
+    is_passive    => '1'
+
+);
+
+isa_ok($ar, "Net::SAML2::Protocol::AuthnRequest");
+
+$override
+    = Sub::Override->override(
+    'Net::SAML2::Protocol::AuthnRequest::issue_instant' =>
+        sub { return 'myissueinstant' });
+
+$xml = $ar->as_xml;
+
+$xp = get_xpath(
+    $xml,
+    samlp => 'urn:oasis:names:tc:SAML:2.0:protocol',
+    saml  => 'urn:oasis:names:tc:SAML:2.0:assertion',
+);
+
+test_xml_attribute_exists($xp, '/samlp:AuthnRequest/@ForceAuthn', 1);
+test_xml_attribute_ok($xp, '/samlp:AuthnRequest/@ForceAuthn', 'true');
+
+test_xml_attribute_exists($xp, '/samlp:AuthnRequest/@IsPassive', 1);
+test_xml_attribute_ok($xp, '/samlp:AuthnRequest/@IsPassive', 'true');
+
+$ar = Net::SAML2::Protocol::AuthnRequest->new(
+    issuer        => 'http://some/sp',
+    destination   => 'http://some/idp',
+    nameid_format => 'urn:oasis:names:tc:SAML:2.0:nameid-format:persistent',
+    nameid_allow_create => 1,
+    force_authn   => '0',
+    is_passive    => '0'
+
+);
+
+isa_ok($ar, "Net::SAML2::Protocol::AuthnRequest");
+
+$override
```

Comment:
I dont think you need this, because $override hasn't been undeffed or got out of scope.

Reply:
true.  I will look